.editorconfig

```diff
@@ -5,7 +5,7 @@ root = true
 insert_final_newline = true
 trim_trailing_whitespace = true
 
-[*.ts, *.vue]
+[*.ts, *.js, *.vue]
```

Comment:
@WoodNeck 
Does it work?

Reply:
It doesn't. I've fixed it in a new commit.

Reply:
commit is added in #288, my mistake.